lobby-db/launch_db

```diff
@@ -1,15 +1,14 @@
 #!/bin/bash
 
-# This script is mostly in part documentation and convenience. Here
-# we tie together the full flow of building a docker image, launching
-# it, and then executing flyway to have a DB schema installed.
-
+# Convenience script to get a local DB up and running on docker
+# with some sample data.
 
 set -eu
 
 path=$(dirname $0)
 
-$path/build_docker
 $path/run_docker
+sleep 5
```

Comment:
Ah the typical "db isn't ready yet" problem with docker ^^

Reply:
Netcat should be installed on every linux system, so this could actually easily be done.
Or you might want to try out something like this: https://github.com/eficode/wait-for which is basically a script that can be used to wait for a service to start

Reply:
Maybe I'll do when I have more time. Currently I can't afford doing things that will take an unknown amount of time, next month this should get a little bit better